.github/workflows/build-and-publish.yml

```diff
@@ -76,6 +77,7 @@ jobs:
           name: workspace
           path: ./
 
+
```

Comment:
Do we need this extra line?

Reply:
Nope. I forgot it was in there.  I had spaced the different jobs out to make it easier to differentiate between them as I was working.  I'll get them all removed.